applications/electron/scripts/after-pack.js

```diff
@@ -42,7 +42,7 @@ const signFile = file => {
 exports.default = async function (context) {
     const running_ci = process.env.BLUEPRINT_JENKINS_CI === 'true';
     const releaseDryRun = process.env.BLUEPRINT_JENKINS_RELEASE_DRYRUN === 'true';
-    const branch = process.env.CHANGE_BRANCH;
+    const branch = process.env.BRANCH_NAME;
```

Comment:
This environment variable exists both for PRs and when merging to master. The previous one only exists for PRs I think. We compare 'branch' to 'master', lower down, as one of the few criteria to proceed with macos signing/notarising.

Reply:
PR build:

![image](https://github.com/eclipse-theia/theia-blueprint/assets/25749063/f2837db7-10c0-4543-83e3-c573c5308dfa)

Merging PR to master (production build):
![image](https://github.com/eclipse-theia/theia-blueprint/assets/25749063/1e7f5795-156e-4f8a-866c-4b4e9d279870)